cli/js/runtime.ts

```diff
@@ -51,7 +49,7 @@ export function start(preserveDenoNamespace = true, source?: string): Start {
   // First we send an empty `Start` message to let the privileged side know we
   // are ready. The response should be a `StartRes` message containing the CLI
   // args and other info.
-  const s = sendSync(dispatch.OP_START);
+  const s = sendSync("op_start");
```

Comment:
Removing `op_` prefix from JS and Rust will be done in follow up PR as I presume?

Reply:
I think I want to keep the op_ prefix for now. It adds clarity.